src/aliceVision/system/ProgressDisplay.cpp

```diff
@@ -0,0 +1,75 @@
+// This file is part of the AliceVision project.
+// Copyright (c) 2011, 2012, 2013 Pierre MOULON
```

Comment:
For new files, you the year of file creation:
// Copyright (c) 2022 AliceVision contributors.

Reply:
Sorry about that. Fixed now.